components/nodemanager-service/Makefile

```diff
@@ -186,6 +186,11 @@ run-db-tests:
 	@POSTGRES_URI="$(POSTGRES_URI)" \
 		go test -v $(shell go list ./pgdb/...) -cover -database
 
+run-one-db-test:
+	@POSTGRES_URI="$(POSTGRES_URI)" \
+		go test -v $(shell go list ./pgdb/...) -run TestRunNodesIntegrationSuite/TestBulkAddNodesAndTagsUpdate -cover -database
+
+
```

Comment:
Did you mean to leave this in here?

Reply:
Yeah... I figured having a target for running a single test would speed up the dev-test cycles for others as well. I don't like that the test is hardcoded inside. Further improvement could be to use git diff to pick up the test that's being worked on and only run that.